package com.example.firstproject.data.DTO.v1;

import com.example.firstproject.model.Role;

public class RegisterDTO {
    private String login;
    private String password;
    private Role role;

    public RegisterDTO(String login, String password, Role role) {
        this.login = login;
        this.password = password;
        this.role = role;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public Role getRole() {
        return role;
    }

    public void setRole(Role role) {
        this.role = role;
    }
}
